### PR TITLE
Remove duplicated files & create subfolders to each kind of file

### PR DESCRIPTION
**Pull request by rafaelks**
Let me know what you guys think about it... I've moved files to `Rocket.Chat.iOS` folder and separated in `Cells`, `ViewControllers` and `Models`.

Also, I removed duplicated `AppDelegate` and `ViewController` files.

### CODE REVIEW

**Comment by gsaslis** [comment]
going through the diff here it shows some (mainly ViewControllers) files as deleted, whereas I'd expect them as moved... ? 
perhaps it's just an issue with the diff making it kinda hard to follow -- are you sure there's no breaking changes ?

File: Rocket.Chat.iOS/ViewController.swift [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-//  ViewController.swift
-//  MMDrawerController
-//
-//  Created by Kornelakis Michael on 8/6/15.
-//  Copyright © 2015 komic. All rights reserved.
-//
-
-import UIKit
-
-class ViewController: UIViewController {
```
Comment (rafaelks) [inline Rocket.Chat.iOS/ViewController.swift]: @gsaslis This is a duplicated file, not being used.

**Comment by gsaslis** [comment]
@kormic it might be best if you have a look at this... 

btw, @rafaelks this branch now has merge conflicts... could you please look into it ?

**Comment by rafaelks** [comment]
@gsaslis Conflicts resolved.

**Comment by gsaslis** [comment]
ok great - I'll leave this up to @kormic to comment on and merge in... ; )

**Comment by kormic** [comment]
Everything looks OK but I want you to let me take another look later today just to be sure...As far I can see it definitely looks a lot cleaner!!!  :)

**Comment by rafaelks** [comment]
@kormic @gsaslis Would be cool have it merged ASAP, because it changes all project structure so we avoid conflicts :-)

**Comment by kormic** [comment]
Yes everything is OK

**Comment by gsaslis** [comment]
Sweet!! let's merge it in then! ; )
